src/main/java/com/softserveinc/ita/pageobjects/modals/AddingAdminModal.java

```diff
@@ -1,4 +1,25 @@
 package com.softserveinc.ita.pageobjects.modals;
 
+import io.qameta.allure.Step;
+
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.ofSeconds;
+
 public class AddingAdminModal extends BaseModal<AddingAdminModal>{
+    @Override
+    public boolean isAddButtonEnabled() {
+        return $x("./span[contains(text(),'Підтвердити')]")
```

Comment:
I'm not sure it will work. Create class variable private final SelenideElement addButton = $x("//span[contains(text(),'Підтвердити')]"); and replace in both methods